app/views/spree/checkout/payment/braintree_vzero/_three_d_secure.html.erb

```diff
@@ -1,218 +1,220 @@
 <script type="text/javascript">
```

Comment:
Looks like indentation for the whole file changed from 2 spaces to 4 spaces.

Reply:
My bad, i'll change it back